### DISCUSSION TITLE
R5/R12 Shunt resistors

### DISCUSSION THREAD

**Discussion by gregcope** [boatybits/boatyFridgePy#6] [General]
HI,

Think you'd mentioned replacing these two with one 0.3R.  Is that correct?

**Comment by boatybits** [comment]
Hi, I took 2 of the resistors off another board and soldered them onto to the main board so it looks like this now , seems to work fine and less power/voltage lost on the way through. Nice to have someone to chat to in here 😊

![https://i.imgur.com/WOQGjtF.png](https://i.imgur.com/WOQGjtF.png)
